python/torch_mlir/extras/onnx_importer.py

```diff
@@ -276,10 +276,13 @@ def import_node(self, node: onnx.NodeProto):
         with InsertionPoint(self._b), Location.name(node.name):
             op_type = node.op_type
             # Handle special op types that materialize to non-op IR constructs.
+            # Handlers return True if the op was handled, else this function
+            # should process it as a general node.
             special_key = f"_handle_node_{op_type}"
             if hasattr(self, special_key):
-                getattr(self, special_key)(node)
-                return
+                wasHandled = getattr(self, special_key)(node)
```

Comment:
Nit: Use snake_case

Reply:
D'oh!  Why isn't my editor doing my thinking for me?!  Fixed